ansible/eda-server
# [AAP-10216] Enable activation in podman env

Enable rulebook activation in Podman environment.

**Test Instruction:**
1. Need to be tested in a VM with podman installed
2. Set environment variables:
```
EDA_DEPLOYMENT_TYPE=podman
EDA_WEBSOCKET_BASE_URL=wss://host.containers.internal
EDA_WEBSOCKET_SSL_VERIFY=no

Optionally you can set
EDA_PODMAN_SOCKET_URL
If you are testing on Mac you can use SSH Tunnel to the Podman Machine and override the url.
```

If testing on a vm the podman socket might not be available intermittently in that case you might want to ensure you set the following for your user. In the example below the user is eda

```
  sudo loginctl enable-linger eda
```

Resolves: [AAP-10216](https://issues.redhat.com/browse/AAP-10216)
